plugin/neovim-fuzzy.vim

```diff
@@ -95,12 +99,12 @@ let s:ag = { 'path': 'ag' }
 function! s:ag.find(root) dict
   return systemlist([
         \ s:ag.path, "--silent", "--nocolor", "-g", "", "-Q"
-        \ ] + (empty(a:root) ? [] : [a:root]))
+        \ ] + (g:fuzzy_hidden ? ["--hidden"] : []) + (empty(a:root) ? [] : [a:root]))
 endfunction
 
 function! s:ag.find_contents(query) dict
   let query = empty(a:query) ? '^(?=.)' : a:query
-  return systemlist(s:ag.path . " --noheading --nogroup --nocolor -S " . shellescape(query) . " .")
+  return systemlist(s:ag.path . (g:fuzzy_hidden ? "--hidden" : "") . "--noheading --nogroup --nocolor -S " . shellescape(query) . " .")
```

Comment:
I think you're missing spaces around the args here, as the string is simply concatinated.

```suggestion
  return systemlist(s:ag.path . (g:fuzzy_hidden ? " --hidden " : " ") . "--noheading --nogroup --nocolor -S " . shellescape(query) . " .")
```